README.md

```diff
@@ -43,22 +71,31 @@ So to summarize:
 3. Type `npm install`
 
 #### Installing webfont dependencies
-Some additional dependencied need to be installed to create a finer icon font.
+Additional system dependencies need to be installed to create a finer icon font.
 Follow the documentation found in https://github.com/sapegin/grunt-webfont
 
 ### Running
 
-Make sure you have `grunt` and `grunt-svgstore` installed (also might require `grunt-svgmin`).
-
-In the commandline, type `grunt`. This will clean up, polish, and generate the following:
+In the commandline, type `$(npm bin)/grunt --verbose`. This will clean up, polish, and generate the following:
 
 - A folder called `svg-min`. This folder contains minimized SVGs of every Social Logo. These SVGs can be dragged directly into Sketch for mockups.
 - A folder called `svg-sprite`. This folder contains a single SVG sprite called `social-logos.svg`, which can be referenced using `use`. But this doesn't work in IE at all yet, eventually it will work in Edge and newer.
 - A folder called `react`. This folder contains a social-logo component for use in React based environment.
 - A folder called `icon-font`. This folder contains different icon font files and css files.
+- A folder called `build`. This folder contains the minified component that will be exported to npm
 
 Do remember to update the React components where they are used, when you add a new icon.
 
+### Publishing to NPM
+
+- Follow install instructions
+- Run `$(npm bin)/grunt --verbose`
```

Comment:
Maybe an npm script would have been a slightly more friendly command to execute?